src/test/java/uk/gov/hmcts/reform/em/orchestrator/endpoint/CcdCloneBundleControllerTest.java

```diff
@@ -0,0 +1,106 @@
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import uk.gov.hmcts.reform.auth.checker.core.service.Service;
+import uk.gov.hmcts.reform.auth.checker.core.service.ServiceRequestAuthorizer;
+import uk.gov.hmcts.reform.auth.checker.core.user.User;
+import uk.gov.hmcts.reform.auth.checker.core.user.UserRequestAuthorizer;
+import uk.gov.hmcts.reform.em.orchestrator.service.caseupdater.CcdBundleCloningService;
+import uk.gov.hmcts.reform.em.orchestrator.service.ccdcallbackhandler.CcdCallbackDto;
+import uk.gov.hmcts.reform.em.orchestrator.service.ccdcallbackhandler.CcdCallbackDtoCreator;
+
+import javax.servlet.http.HttpServletRequest;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@AutoConfigureMockMvc
+public class CcdCloneBundleControllerTest {
+
+    @MockBean
+    private CcdCallbackDtoCreator ccdCallbackDtoCreator;
+
+    @MockBean
+    private CcdBundleCloningService ccdBundleCloningService;
+
+    @MockBean
+    private ServiceRequestAuthorizer serviceRequestAuthorizer;
+
+    @MockBean
+    private UserRequestAuthorizer userRequestAuthorizer;
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Before
+    public void setupMocks() {
+        Mockito
+                .when(serviceRequestAuthorizer.authorise(Mockito.any(HttpServletRequest.class)))
+                .thenReturn(new Service("ccd"));
+
+        Mockito
+                .when(userRequestAuthorizer.authorise(Mockito.any(HttpServletRequest.class)))
+                .thenReturn(new User("john", Stream.of("caseworker").collect(Collectors.toSet())));
+
+        Mockito
+                .when(ccdCallbackDtoCreator.createDto(Mockito.any(HttpServletRequest.class), Mockito.any(String.class)))
+                .thenReturn(new CcdCallbackDto());
+    }
+
+    @Test
+    public void shouldCallCcdCallbackHandlerService() throws Exception {
+
+        this.mockMvc
+                .perform(post("/api/clone-ccd-bundles")
+                        .content("[]")
+                        .contentType(MediaType.APPLICATION_JSON)
+                        .header("Authorization", "xxx")
+                        .header("ServiceAuthorization", "xxx"))
+                .andDo(print()).andExpect(status().isOk());
+
+        Mockito
+                .verify(ccdBundleCloningService, Mockito.times(1))
+                .updateCase(Mockito.any(CcdCallbackDto.class));
+    }
+
+    @Test
+    public void shouldCallCcdCallbackHandlerServiceUpdateException() throws Exception {
```

Comment:
In the longer term I think we should move away from these tests as they are too close to the functional tests